packages/crayons-core/src/components/select-option/select-option.tsx

```diff
@@ -214,7 +227,33 @@ export class SelectOption {
     );
   }
 
+  createConversationDescription() {
+    const authorDetails = [];
+    if (this.author.name) authorDetails.push(this.author.name);
+    if (this.author.email) authorDetails.push(this.author.email);
+    if (this.author.phone) authorDetails.push(this.author.phone);
+
+    return this.subText ? (
+      <div class={'description ' + 'icon-margin '}>
+        <span class='description-text'>{this.text}</span>
+        <span class='description-subText-conversation'>{this.subText}</span>
+        <span class='description-author-details'>
+          {authorDetails?.join(' | ')}
+        </span>
+      </div>
+    ) : (
+      <span class={'description ' + 'icon-margin'}>{this.text}</span>
+    );
+  }
+
   createIcon() {
+    const { imageSrc } = this.graphicsProps;
+
+    if (imageSrc) {
+      return (
+        <img src={imageSrc} class='image-icon-dimension-standard' alt='icon' />
```

Comment:
@harshith-venkatesh-freshworks Why do we need an imageSrc here? Can you send it as an svg to data-svg so this can be rendered in fw-icon?

Reply:
So, we would be depending on the product here. They have mentioned there would be image source for few icons like whatsapp, instagram icons etc., leading to this approach